core/src/validator.rs

```diff
@@ -285,19 +284,6 @@ impl Validator {
             }
         }
 
-        info!("Cleaning accounts paths..");
```

Comment:
Should we keep this cleanup here as well for other users of `Validator`, such as `solana-test-validator` and `local-cluster`?

Reply:
`local-cluster` doesn't really need it although better to test that path there than not, I'm not sure about `solana-test-validator`.

The other option is move all the snapshot download stuff into `Validator::new` and then re-order it there.

Reply:
`solana-test-validator` probably doesn't strictly need it I guess, the accounts are generally expected to be small here since it's just a dev tool.   Moving the snapshot stuff into Validator::new would be nice, probably a ton of churn with low ROI at this point.  One day I'd love to Validator::new to live in `validator/` at which point download refactoring would be much easier.

Reply:
Ok, I'll close for now, it was to mitigate validators having to rm -rf this directory on startup manually.